share/docker/Dockerfile

```diff
@@ -71,10 +71,6 @@ RUN apt-get -qqy update \
                                                  python3.10 \
                                                  python3.11 \
                                                  python3.11-dev \
```

Comment:
Is it time to add 3.12 here ?

Reply:
Given the fact that 3.13 has been released, I think it does the time.